doc/en/fcrontab.1.sgml

```diff
@@ -53,33 +53,41 @@ A copy of the license is included in gfdl.sgml.
-	    directory (<filename>&fcrontabsdir;</filename>) to allow future
-	    editions, and a formatted file is generated for the &fcron; daemon,
-	    which is signaled once about ten seconds before the next minute for
-	    all changes made previously. The daemon is not informed of the changes
-	    immediately but at most once a minute
+	    directory (<filename>&fcrontabsdir;</filename>) to allow it to be
+	    edited, and a formatted file is generated for the &fcron; daemon.
+		This formatted (non human readable) file with all recent changes
+		included is then sent to the daemon.
+		The daemon is not informed of the changes immediately but after some time
 	    to keep ill disposed users from blocking the daemon by installing
-	    &fcrontabf;s over and over (ie. denial of service attack). We will call
+	    &fcrontabf;s over and over (i.e. denial of service attack).
+		By default, the file is sent once per minute (about ten seconds before
+		the full minute) to the daemon, the parameter
+		<option>--with-max-fcrontab-reload-delay-seconds</option>
+		can be used to specify a different maximum time to send this data.
+		We will call
 	    "<emphasis>fcrontab</emphasis>" the source file of the &fcrontabf;
-	    in the following.</para> <para>A user can install a &fcrontabf; if
-	    he is listed in the <filename>&etc;/&fcron.allow;</filename> and not
-	    (unless by the keyword all) listed in
+	    in the following.</para>
+		<para>The fcrontab files that a user modifies has to follow the format
+		defined in <link linkend="fcrontab.5">&fcrontab;(5)</link> .</para>
+		<para>Users can install a &fcrontabf; if
+	    they are listed in <filename>&etc;/&fcron.allow;</filename> and not listed in
 	    <filename>&etc;/&fcron.deny;</filename> (see section "<link
-	    linkend="fcrontab.1.files">files</link>" below). If neither
+	    linkend="fcrontab.1.files">files</link>" below).
+		The keyword 'all' is a shorthand for all users. If neither
 	    <filename>fcron.allow</filename> nor <filename>fcron.deny</filename>
 	    exist, all users are allowed. None of these files have to exist, but
 	    if they do, the deny file takes precedence.</para> <para>The first
 	    form of the command is used to install a new &fcrontabf; file, from
-	    any named file or from standard input if the pseudo-filename "-" is
-	    given, replacing the previous one (if any): each user can have only
+	    any named file or from standard input (if the pseudo-filename "-" is
+	    given), replacing the previous one (if any). Each user can have only
 	    one &fcrontabf;.</para> <para>For instance, root can create a
 	    systemwide fcrontab file, say <filename>/etc/fcrontab</filename>,
 	    and run "<command>fcrontab</command>
 	    <filename>/etc/fcrontab</filename>" to install the new version after
-	    each change of the file. Or (s)he can create a new fcrontab running
+	    each change of the file. Or root can create a new fcrontab running
```

Comment:
not "root" -- did you mean "they" ?

Reply:
no, I meant root as this paragraph is about root. In the line befor it says `... root can create ...`, so this is again `root` doing stuff here.
But if you insist, I'll reword to whatever you think is better.
But honestly, it feels like gold plating here -- I think either is fine ;)